test/e2e/landing-page.spec.ts

```diff
@@ -26,7 +26,7 @@ test.describe('Landing Page', () => {
     await page
       .getByRole('paragraph')
       .filter({
-        hasText: 'DNS Record: Lorem Ipsum is simply dummy text of the printing and typesetting ind',
+        hasText: 'DNS Record is a type of data stored in Domain Name System that maps a domain',
```

Comment:
"DNS Record is a data stored in Domain Name System (DNS) servers, which maps a value to a domain name"